### DISCUSSION TITLE
`npm run create docs/*` adds invalid images?

### DISCUSSION THREAD

**Discussion by OCram85** [thuliteio/doks#221] [Q&A]
It seems like the archetype for docs always adds the shortcode for images:
` {{< img src="{{ .Name | urlize }}.jpg" alt="{{ replace .Name "-" " " | title }}" caption="{{ replace .Name "-" " " | title }}" class="wide" >}}`

This seems to lead to an error when disabling the draft mode.
What does this tag supposed to do? - Did I miss anything?

Example:
- run `npm run create docs/archive/rules2.md`
- the `rules2.md` content is:

```markdown
---
title: "Rules2"
description: ""
lead: ""
date: 2021-04-07T14:36:28+02:00
lastmod: 2021-04-07T14:36:28+02:00
draft: true
images: []
menu: 
  docs:
    parent: ""
weight: 999
toc: true
---

{{< img src="rules2.jpg" alt="Rules2" caption="Rules2" >}}
```

- disable the `draft: true` tag

![image](https://user-images.githubusercontent.com/13993700/113868139-a21fdc00-97af-11eb-9653-31afed9c9b92.png)

**Comment by h-enk** [comment]
Good point. The idea is to generate a page specific image short code automatically (so you don't have to add it manually). It's meant to be there for your convenience. But yes, you should have a corresponding image in your page bundle, when setting `draft: true`.

I must admit, more often than not, I delete the short code myself immediately (for I don't use it).

Since it could lead to confusion, I will then remove the short codes from the archetype files. What do you think?

**Reply by OCram85** [reply]
I think removing the short cote from the archetype would work for most users. But that's only my opinion ^^
